Sources/ProjectDescription/SettingsTransformers.swift

```diff
@@ -0,0 +1,137 @@
+import Foundation
+
+public extension SettingsDictionary {
+    mutating func merge(_ other: SettingsDictionary) {
+        merge(other) { $1 }
+    }
+
+    func merging(_ other: SettingsDictionary) -> SettingsDictionary {
+        merging(other) { $1 }
+    }
+}
+
+private extension SettingValue {
+    init(_ string: String) {
+        self = .init(stringLiteral: string)
+    }
+
+    init(_ bool: Bool) {
+        self = .init(booleanLiteral: bool)
+    }
+}
+
+public enum SwiftCompilationMode: String {
+    case singlefile
+    case wholemodule
+}
+
+public enum SwiftOptimizationLevel: String {
+    case o = "-O"
+    case oNone = "-Onone"
+    case oSize = "-Osize"
+}
+
+public extension SettingsDictionary {
+    // MARK: - Deployment target and SDKROOT
+
+    /// Sets "SDKROOT" to `value`
+    func sdkRoot(_ value: String) -> SettingsDictionary {
+        merging(["SDKROOT": SettingValue(value)])
+    }
```

Comment:
Is there still a need for this one? (Tuist automatically sets this based on the Target's platform).

https://github.com/tuist/tuist/blob/master/Sources/TuistGenerator/Generator/ConfigGenerator.swift#L183